src/filebuilders/file_builder.py

```diff
@@ -1,31 +1,63 @@
 import abc
+import os
 from datetime import datetime
 
 
 class FileBuilder(abc.ABC):
 
-    def __init__(self, google_drive_connector): 
-        self.google_drive_connector = google_drive_connector
+    def __init__(self, google_drive_connector, domain_object_config):
+        file_name = domain_object_config['file_name']
+        file_extension = domain_object_config['file_builder_name'].lower()
+
+        self.__google_drive_connector = google_drive_connector
+        self.__file_name = file_name + '_{}.' + file_extension
```

Comment:
This syntax choice seems a little... odd? maybe can be written prettier to be readable to what the name will be being defined as